draft-irtf-cfrg-frost.md

```diff
@@ -305,6 +305,12 @@ and
 H2(m) = H(contextString || "chal" || len(m) || m)
 ~~~
 
+and 
+
+~~~
+H3(m) = H(m)
```

Comment:
I would add a comment here that a domain seperator is omitted specifically for backwards compatibility and citation.